packages/edit-navigation/src/components/editor/list-view.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { Spinner } from '@wordpress/components';
+import { __experimentalBlockNavigationTree } from '@wordpress/block-editor';
+
+export default function ListView( { isPending, blocks } ) {
+	const [ selectedBlockId, setSelectedBlockId ] = useState(
+		blocks[ 0 ]?.clientId
```

Comment:
To be honest I'm not even sure why we're using local state instead of `select( 'core/block-editor' ).getSelectedBlockClientId()`. Maybe @talldan knows?

Reply:
It was already this way in `master`, so I've made a note to come back to this later.

Reply:
I think this was @adamziel's change, https://github.com/WordPress/gutenberg/issues/22705 should be the issue relating to the code.